app/views/layouts/_navbar.html.erb

```diff
@@ -260,7 +261,11 @@
       <% @breadcrumbs&.each do |b| %>
         <span class="item"><%= b %></span>
       <% end %>
-      <% if @title %>
+      <% if @extra_title %>
```

Comment:
Tried using breadcrumbs but the issue is that breadcrumbs don't register as the title of the tab in browser. If I make the title a link, it would show the entire link as the title and if I don't then file manager would not be clickable which is not ideal.

Reply:
Okay, fixed.